src/main/groovy/edu/oregonstate/mist/cardsapi/db/CardFluent.groovy

```diff
@@ -0,0 +1,100 @@
+package edu.oregonstate.mist.cardsapi.db
+
+import edu.oregonstate.mist.cardsapi.core.Card
+import edu.oregonstate.mist.cardsapi.mapper.CardsMapper
+import org.skife.jdbi.v2.DBI
+import org.skife.jdbi.v2.Query
+import org.skife.jdbi.v2.Handle
+
+class CardFluent {
+
+    private DBI dbi
+
+    CardFluent(DBI dbi) {
+        this.dbi = dbi
+    }
+
+    // Converts list of strings to comma-separated list in SQL
+    static String listToSql (List<String> list) {
+        String str = "("
+        for(int i = 0; i < list.size() - 1; i++) {
+            str += "\'" + list[i] + "\',"
+        }
+        str += "\'" + list[list.size() - 1] + "\')"
+        str
+    }
+
+    // Builds list of LIKE statements for each string in keywords
+    static String keywordsToSql (List<String> keywords) {
+        String str = ""
+        for(int i = 0; i < keywords.size(); i++) {
+            str += "AND LOWER(DESCRIPTION) LIKE LOWER('%'||"
+            str += "\'" + keywords[i] + "\'"
+            str += "||'%')"
+        }
+        str
+    }
+
+    List<Card> getCards(List<String> types,
+                        String name,
+                        List<String> colors,
+                        List<String> rarities,
+                        Integer energyMin,
+                        Integer energyMax,
+                        List<String> keywords,
+                        Integer cardNumber,
+                        Boolean isRandom) {
+        Handle h = dbi.open()
+        String query = """
+        
+        SELECT *
```

Comment:
I'd advise doing a `select *` in code like this. Instead, we should be referencing the specific columns.